torch/_dynamo/variables/misc.py

```diff
@@ -237,9 +253,28 @@ def call_method(
             try:
                 attr = attr.as_python_constant()
             except NotImplementedError:
-                unimplemented(f"non-const delattr attr: {attr}")
+                unimplemented_v2(
```

Comment:
you don't want to attach the cause *from* the exception on line 255?

Reply:
Thanks for your review. Have updated.